src/main/service/tableService.ts

```diff
@@ -0,0 +1,57 @@
+import moment from 'moment';
+import { SubscriptionActions } from '../resources/actions/subscriptionActions';
+
+export class TableService {
+  subscriptionActions = new SubscriptionActions();
+  
+  generateCaseTableRows(userId: number): any[] {
```

Comment:
can these methods take in data thats returned from api instead? so you dont have to do the call here, because then youre still limiting this method to only this situation, so something like: 
```
generateFourColumnTable(data)
```
then you can remove line 8 + 10 and this method is focused on creating a table with 4 columns